packages/hydrogen/src/logger.ts

```diff
@@ -0,0 +1,103 @@
+import {yellow, red} from 'kolorist';
+
+type LoggerMethod = (
+  context: {request: Request},
+  ...args: Array<any>
+) => void | Promise<any>;
```

Comment:
`Promise<void>` or do we actually expect to get something out of the promise here?